engine/apps/api/serializers/alert_group_escalation_snapshot.py

```diff
@@ -0,0 +1,55 @@
+from rest_framework import serializers
+
+from apps.api.serializers.custom_button import CustomButtonFastSerializer
+from apps.api.serializers.escalation_policy import EscalationPolicySerializer
+from apps.api.serializers.schedule_base import ScheduleFastSerializer
+from apps.api.serializers.user import FastUserSerializer
+from apps.api.serializers.user_group import UserGroupSerializer
+from apps.api.serializers.webhook import WebhookFastSerializer
+
+
+class EscalationPolicySnapshotAPISerializer(EscalationPolicySerializer):
+    """Serializes AlertGroup escalation policies snapshots for API endpoint"""
+
+    notify_to_users_queue = FastUserSerializer(many=True, read_only=True)
+    notify_schedule = ScheduleFastSerializer(read_only=True)
+    notify_to_group = UserGroupSerializer(read_only=True)
+    custom_button_trigger = CustomButtonFastSerializer(read_only=True)
+    custom_webhook = WebhookFastSerializer(read_only=True)
```

Comment:
_nit_, do you need to include `read_only` here if these fields are also included in `Meta.read_only_fields`?

Reply:
There is (or was?) an issue in DRF, when `read_only_fields` only works for fields which are not explicitly defined in the Serializer